map_test.go

```diff
@@ -111,7 +111,7 @@ func TestMapPin(t *testing.T) {
 	// Issue 51: pad path out to a power of two, to avoid having a
 	// trailing zero at the end of the allocation which holds the string.
 	path := tmp + string(filepath.Separator)
-	path += strings.Repeat("a", 32-len(path))
+	path += strings.Repeat("a", 64-len(path))
```

Comment:
Why did this change?

Reply:
I will drop this change, but this test failed because  ioutil.TempDir("/sys/fs/bpf", "ebpf-test") + "/" is already 32 bytes (chars) long
I would need to check old kernel behaviour for this test, and may skip it on recent kernels, it's another story as some others tests are failing as well (not related to this PR or previous one)